src/TracingDriverManager.php

```diff
@@ -22,7 +22,6 @@ class TracingDriverManager extends Manager
     public function __construct($app)
     {
         parent::__construct($app);
-        $this->config = $this->app->make('config');
```

Comment:
Please, replace the deleted line with 
`$this->config = $this->container->make('config');
@iyhunko